docs/content/en/docs/implementing/evaluations.md

```diff
@@ -3,3 +3,85 @@ title: Evaluations
 description: Understand Keptn evaluations and how to use them
 weight: 150
 ---
+A `KeptnEvaluation` is a `KeptnMetric` that has a defined target value.
```

Comment:
```suggestion
A `KeptnEvaluation` allows to check wether a `KeptnMetric` has a defined target value.
```

Reply:
Alas, "allows to check" is not valid English syntax.  It's a real shame, I know.  So I reworded this slightly -- see if this works